packages/instrumenter/package.json

```diff
@@ -41,13 +41,15 @@
     "@stryker-mutator/api": "6.3.1",
     "@stryker-mutator/util": "6.3.1",
     "angular-html-parser": "~3.0.0",
+    "svelte": "^3.52.0",
```

Comment:
Adding this here will increase the footprint of installing Stryker. Can we make it a `devDependency` instead? 

This would mean that end users should install `svelte` themselves if they want to mutate `.svelte` files, which is reasonable. However, these import statements will _fail_ as a result: `import ... from 'svelte'`. Instead of using import statements, we should use import _expressions_. I will place a remark at one of those files to make it clear.